test/expected/build/application.i/dest/Component-preload.js

```diff
@@ -3,9 +3,11 @@ jQuery.sap.registerPreloadedModules({
 "version":"2.0",
 "modules":{
 	"application/i/Component.js":function(){sap.ui.define(["sap/ui/core/UIComponent"],function(n){"use strict";return n.extend("application.i.Component",{metadata:{manifest:"json"}})});
+//# sourceMappingURL=Component.js.map
```

Comment:
Should those comments be part of the preload? If the browser ignores them I guess it's okay. If it causes problems, we should remove them.

Reply:
I *think* they are not a problem unless they are at the end of the file. Anyways, the source map change for the module bundler will remove them 👍